Test failures building 1.6.0 (x86_64 linux)

```
FAIL	github.com/grafana/loki/pkg/chunkenc [build failed]
```
```
    drop_test.go:373: validateDropConfig() error = drop stage invalid duration, 10y cannot be converted to a duration: time: unknown unit "y" in duration "10y", wantErr = drop stage invalid duration, 10y cannot be converted to a duration: time: unknown unit y in duration 10y
    --- FAIL: Test/invalid_idle_duration (0.00s)
```
```
FAIL	github.com/grafana/loki/pkg/logentry/stages	0.028s
```

--- a/pkg/logentry/stages/drop_test.go
+++ b/pkg/logentry/stages/drop_test.go
@@ -342,7 +342,7 @@ func Test_validateDropConfig(t *testing.T) {
 			config: &DropConfig{
 				OlderThan: &dropInvalidDur,
 			},
-			wantErr: fmt.Errorf(ErrDropStageInvalidDuration, dropInvalidDur, "time: unknown unit y in duration 10y"),
+			wantErr: fmt.Errorf(ErrDropStageInvalidDuration, dropInvalidDur, "time: unknown unit \"y\" in duration \"10y\""),
 		},
 		{
 			name: "Invalid Config",

--- a/pkg/logentry/stages/metrics_test.go
+++ b/pkg/logentry/stages/metrics_test.go
@@ -222,7 +222,7 @@ func Test(t *testing.T) {
 					IdleDuration: &metricTestInvalidIdle,
 				},
 			},
-			errors.Errorf(ErrInvalidIdleDur, "time: unknown unit f in duration 10f"),
+			errors.Errorf(ErrInvalidIdleDur, "time: unknown unit \"f\" in duration \"10f\""),
 		},
 		"valid": {
 			MetricsConfig{

